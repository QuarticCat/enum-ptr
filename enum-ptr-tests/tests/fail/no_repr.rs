#![allow(dead_code)]

use enum_ptr::EnumPtr;

#[derive(EnumPtr)]
enum Foo<'a, 'b> {
    A(&'a i32),
    B(&'b i32),
}

fn main() {}
