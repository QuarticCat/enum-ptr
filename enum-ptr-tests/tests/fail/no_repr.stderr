error: EnumPtr requires `#[repr(C, usize)]`
 --> $DIR/no_repr.rs:6:6
  |
6 | enum Foo<'a, 'b> {
  |      ^^^
